doordeck-sdk/src/commonMain/kotlin/com/doordeck/multiplatform/sdk/cache/CapabilityCache.kt

```diff
@@ -0,0 +1,22 @@
+package com.doordeck.multiplatform.sdk.cache
+
+import com.doordeck.multiplatform.sdk.api.model.CapabilityStatus
+import com.doordeck.multiplatform.sdk.api.model.CapabilityType
+
+internal object CapabilityCache {
+    private val cache = mutableMapOf<String, Map<CapabilityType, CapabilityStatus>>()
```

Comment:
Is thread safety going to be an issue here?

Reply:
In theory, yes, but so far, I have been unable to reproduce it with a test, so I’m not entirely sure.